src/app/app.component.scss

```diff
@@ -1,7 +1,4 @@
-h1,a {
-    margin: 10px 10px 10px 10px;
+.font-weight-bold,
+.text-warning {
```

Comment:
Still could use margin classes from bootstrap

Reply:
Also if those classes are from bootstrap, do not change them. Add a new class, your own and add it to the html element.